app/views/podcasts/index.html.erb

```diff
@@ -1,35 +1,61 @@
-<div class="jumbotron-jumbotron-fluid">
-  <h1 class="display-4">Hello, world!</h1>
-  <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
+<div class="jumbotron-jumbotron-fluid center-search">
+  <h1 class="display-4">Podsmack</h1>
+  <p class="lead">Your next favorite podcast is waiting to be found!</p>
   <hr class="my-4">
-  <p>It uses utility classes for typography and spacing to space content out within the larger container.</p>
-  <a class="btn btn-primary btn-lg" href="#" role="button">Learn more</a>
+  <p>Select exactly what you're looking for.</p>
 </div>
 <br>
 <br>
+<br>
+
+
+
 <%= form_tag(podcasts_path, method: "get") do %>
-  <%= label_tag 'Location' %>
-  <%= select_tag(:location, options_for_select([nil, 'Denver', 'NYC', 'Dallas', 'San Francisco']), { class: "btn btn-outline btn-xl js-scroll-trigger" })%>
+  <div class="container">
+    <div class="row">
 
-  <%= label_tag :tags %>
-  <%= collection_check_boxes(:tags, :ids, Tag.all, :id, :name, include_hidden: false) %> <br>
+        <div class="col-sm center-search">
```

Comment:
This creates individual columns for the page. Getting them to cooperate was very painful.